<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>790</width>
      <height>425</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>790</width>
      <height>425</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>790</width>
      <height>425</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="TabRaw">
     <attribute name="title">
      <string>CONTROL</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonInhibit">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>290</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>INHIBIT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRun">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>290</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>RUN</string>
      </property>
     </widget>
     <widget class="QSlider" name="SliderRailSpeed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>771</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 40px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
    border: 1px solid #5c5c5c;
    width: 50px;
    margin: -5px 0 ; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 3px;
}</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="SliderFocusSPSpeed_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>771</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 40px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
    border: 1px solid #5c5c5c;
    width: 50px;
    margin: -5px 0 ; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 3px;
}</string>
      </property>
      <property name="minimum">
       <number>50</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>90</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAlclr">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>ALCLR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonReverse">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>REVERSE</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdRawSpeed">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdRawDistance">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ADJUST ES</string>
     </attribute>
     <widget class="QSlider" name="SliderStopDistance">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>621</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 40px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
    border: 1px solid #5c5c5c;
    width: 50px;
    margin: -5px 0 ; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 3px;
}</string>
      </property>
      <property name="minimum">
       <number>50</number>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdStopDistance">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>290</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEs2Set">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>150</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>ES 2 SET</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEs1Set">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>ES 1 SET</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEsDisable">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>ES DISABLE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEsEnable">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>150</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>ES ENABLE</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdRaw2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdRaw1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdRaw4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdRaw3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>MISC</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonApplicationExit">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>320</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>APP EXIT</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelInhibit">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>INH</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelRev">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>REV</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelES2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>ES2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelES1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>ES1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelRun">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelConnected">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>CON</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
